packages/xo-web/src/common/editable/index.js

```diff
@@ -341,74 +332,80 @@ export class Number extends Component {
   }
 }
 
-export class Select extends Editable {
+class SimpleSelect_ extends Editable {
   static propTypes = {
-    options: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
-      .isRequired,
     renderer: PropTypes.func,
-  }
-
-  componentWillReceiveProps(props) {
-    if (
-      props.value !== this.props.value ||
-      props.options !== this.props.options
-    ) {
-      this.setState({
-        valueKey: findKey(props.options, option => option === props.value),
-      })
-    }
+    value: PropTypes.oneOfType([PropTypes.oneOf([null]), PropTypes.object]),
   }
 
   get value() {
-    return this.props.options[this.state.valueKey]
-  }
-
-  _onChange = event => {
-    this.setState({ valueKey: getEventValue(event) }, this._save)
+    return this.state.value
   }
 
-  _optionToJsx = (option, key) => {
-    const { renderer } = this.props
+  _onChange = value => this.setState({ value }, this._save)
 
+  _renderDisplay() {
+    const { children, renderer, value } = this.props
     return (
-      <option key={key} value={key}>
-        {renderer ? renderer(option) : option}
-      </option>
+      children || (
+        <span>{renderer !== undefined ? renderer(value) : value}</span>
```

Comment:
`value` is an object so it can't be rendered as is.
I suggest:
```js
children ||
  <span>{renderer !== undefined
    ? renderer(value)
    : value !== undefined
    ? value.label
    : ' ' // ensures that the input is clickable
  }</span>
```